pkg/aqua/command_test.go

```diff
@@ -30,6 +30,7 @@ func TestCommand_Scan(t *testing.T) {
 		ScannerCLIShowNegligible:              true,
 		ScannerCLIOverrideRegistryCredentials: true,
 		ScannerCLIDirectCC:                    true,
+		ScannerCLIRegisterImages:              etc.Compliant,
```

Comment:
Thought: I wonder if this is critical enough to warrant a full table driven test but at the same time I don't know of a good way to handle so many options/combinations.

Reply:
Me neither. That's why I'm just testing a happy path here. Anyway if there's something wrong with arguments passed to scannercli we'll see a very clear error message. But yeah, that's a good point to see if we can do better.